rocAL/rocAL_pybind/examples/rocAL_api_coco_pipeline.py

```diff
@@ -58,170 +57,169 @@ def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False,
+        path = "OUTPUT_IMAGES_PYTHON/COCO_READER/"
         isExist = os.path.exists(path)
         if not isExist:
             os.makedirs(path)
     except OSError as error:
         print(error)
-    image_path = sys.argv[1]
-    annotation_path = sys.argv[2]
-    device = sys.argv[3]
-    _rali_cpu = True if device == "cpu" else False
-    batch_size = int(sys.argv[4])
-    num_threads = 1
-    device_id = 0
-    random_seed = random.SystemRandom().randint(0, 2**32 - 1)
-    crop=800
-    local_rank = 0
-    world_size = 1
-    # Anchors - load default anchors from a binary file  
-    with open('Default_anchors_retinanet.bin', 'rb') as f_read:
-        anchor_bytes = f_read.read()
-    num_anchors = len(anchor_bytes) // 4
-    anchor_list = []
-    for i in range(num_anchors):
-        anchor_bytes_single = anchor_bytes[i * 4: (i + 1) * 4]
-        anchor_float = struct.unpack('!f', anchor_bytes_single)[0]
-        anchor_list.append(anchor_float)
-    print("*********************************************************************")
-    coco_train_pipeline = Pipeline(batch_size = batch_size, num_threads = num_threads, device_id = device_id,seed = random_seed, rocal_cpu = _rali_cpu, tensor_layout = types.NCHW)
-    with coco_train_pipeline:
-        jpegs, bboxes, labels = fn.readers.coco(file_root=image_path,
-                                                 annotations_file=annotation_path, 
-                                                 random_shuffle=True,
-                                                 shard_id=local_rank, 
-                                                 num_shards=world_size,
-                                                 seed=random_seed, 
-                                                 is_box_encoder=False,
-                                                 is_box_iou_matcher=True)
-
-        print("*********************** SHARD ID ************************",local_rank)
-        print("*********************** NUM SHARDS **********************",world_size)
-        images_decoded = fn.decoders.image(jpegs, output_type = types.RGB, file_root=image_path, annotations_file=annotation_path, random_shuffle=False,shard_id=local_rank, num_shards=world_size)
+
+    # Create Pipeline instance
+    pipe = Pipeline(batch_size=batch_size, num_threads=num_threads, device_id=args.local_rank,
+                    seed=random_seed, rocal_cpu=rocal_cpu, tensor_layout=tensor_format, tensor_dtype=tensor_dtype)
+    # Use pipeline instance to make calls to reader, decoder & augmentation's
+    with pipe:
+        jpegs, bboxes, labels = fn.readers.coco(annotations_file=annotation_path)
+        images_decoded = fn.decoders.image(jpegs, output_type=types.RGB, file_root=image_path,
+                                           annotations_file=annotation_path, random_shuffle=False, shard_id=local_rank, num_shards=world_size)
+        res_images = fn.resize(images_decoded, resize_width=300, resize_height=300)
```

Comment:
@SundarRajan28 why do we have so many augmentations to test coco reader? I do not see it previously

Reply:
The coco pipeline example is changed to be similar to the ToT example. It has the same number of augmentations as this example.